openreview/venue/process/recruitment_pre_process.js

```diff
@@ -0,0 +1,37 @@
+async function process(client, edit, invitation) {
+
```

Comment:
Not sure if you want to add this:
```suggestion
  client.throwErrors = true

```